### ISSUE TITLE
Datepicker accessibility

### ISSUE THREAD

**Issue by jbickar** [SU-SWS/stanford_events_importer#43]
For admin forms and Views exposed filters. Datepicker should have keyboard access and focus. 

See the Assets date picker: http://assets.cms.gov/resources/framework/2.0/Pages/index.html (implemented on events.stanford.edu)

First, get a review of the accessibility of current date picker. I cannot get keyboard access to the one on events.stanford.edu either, so there's something wrong with my testing.
